plugins/google_analytics/girder_google_analytics/web_client/main.js

```diff
@@ -5,21 +5,24 @@ import './lib/backbone.analytics';
 import './routes';
 
 events.on('g:appload.after', function () {
```

Comment:
To minimize the amount of change, what about adding:

```
var dnt = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
if (dnt == "1" || dnt == "yes") {
  return;
}
```

Reply:
I like change :-)... sometimes... ;-).